### PR TITLE
Menu Toggle script

### PR DESCRIPTION
**Pull request by Aliahmad-karimi**
-add the menu and close the icon
-modify the properties for the menu

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+menuBar.addEventListener('click', () => {
+  menuBar.classList.toggle('active');
+  toolSocial.classList.toggle('active');
+});
+
+document.querySelectorAll('.Line').forEach(n, () => n.addEventListener('click', () => {
+  menuBar.classList.remove('active');
+  toolSocial.classList.remove('active');
+}));
```
Comment (Bondok6) [inline script.js]: - [ ] When I click on the `exit-icon` or the links the mobile menu should disappear, but nothing happened (Still appear), Kindly fix it 👌

image

File: style.css [hunk 2]
```diff
@@ -45,23 +51,94 @@ body {
...
+  list-style-type: none;
+  background-color: #2c3fefc9;
+  backdrop-filter: blur(2px);
+  margin-top: -30px;
+}
+
+.main-page .header .tools .tool-social.active {
+  display: block;
+  margin-left: -62%;
+}
```
Comment (Bondok6) [inline style.css]: - [ ] Please make sure that the width of the mobile list is equal to the width of the entire screen 👌 ِAccording to the requirements You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) 💯

| Your Design  | Figma Design  |   
| --- | --- |
|   |  |

> To fix this issue I can add `left: 0;` to this class `.main-page .header .tools .tool-social`

and remove the `margin-left` from here 😉
```
.main-page .header .tools .tool-social.active {
  display: block;
  margin-left: -62%;
}
```

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @Aliahmad-karimi and @belmeetmule  🙋‍♂️, 

Good job so far! 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✅
- Your project is professional ✅
- Your repo is professional ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @Aliahmad-karimi, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
 
> ## [Optional] suggestions:
>- [ ] Next time, and for future Pull Requests, I suggest you please kindly add a more descriptive PR title.
> See a demo below:
pull_request_desc
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**